src/cmake/config/linux-gcc.txt

```diff
@@ -66,6 +66,7 @@ set (c_compiler_flags_common
     -Wno-sign-compare
     -Wno-unused-function
     -Wno-reorder
+    -Wno-strict-aliasing
```

Comment:
Interesting; we really shouldn't have those I think. Can you show me one example where we have this problem in our own code?